### ISSUE TITLE
Bleeding of the text - on sign in page 

### ISSUE THREAD

**Issue by magdalenadrafiova** [DMPRoadmap/roadmap#2949]
It has been pointed out that when the resolution of the screen is anything bigger than 100% the text does not align with the box, see below: 
<img width="456" alt="Screenshot 2021-06-17 at 16 11 38" src="https://user-images.githubusercontent.com/47178106/122424843-f05b1380-cf86-11eb-973a-c0ebb1d61318.png">

Could we fix so it fits nicely in any size?

**Comment by briri** [comment]
Should be a fairly easy patch.

There is a larger issue overall with the responsiveness of the site. We're 2 major versions behind on Bootstrap (we are on 3.4 and current is 5.x) as well. We should consider upgrading to the latest (very large undertaking though).
